test/Project.toml

```diff
@@ -11,7 +11,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 [compat]
 Aqua = "0.8"
 MeshIntegrals = "0.13"
```

Comment:
```suggestion
```
This is not related to this PR, but I think you don't need MeshIntegrals.jl in the Project.toml in test/ (you can also delete it in the [deps] section, but GitHub doesn't allow me to comment there). See also the comment here: https://docs.julialang.org/en/v1/stdlib/Test/#Testing-Our-Package

Reply:
Done.

Reply:
Thanks!